docs/source/release/v6.2.0/direct_geometry.rst

```diff
@@ -25,6 +25,7 @@ CrystalField
 Improvements
 ############
 - Added documentation and warning messages in the :ref:`Crystal Field Python Interface` related to IntensityScaling
+- Exposed method ''functionDeriv'' to the python interface
```

Comment:
I guess you want this here as well, but the change feels more relevant to the framework.rst file.

Reply:
Good point. I will just move it to framework then.